rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -1286,6 +1286,7 @@ namespace np
 		ret.cb_arg = (optParam && optParam->cbFuncArg) ? optParam->cbFuncArg : ctx->default_match2_optparam.cbFuncArg;
 		ret.cb     = (optParam && optParam->cbFunc) ? optParam->cbFunc : ctx->default_match2_optparam.cbFunc;
 		ret.event_type = event_type;
+		ret.abortable = abortable;
```

Comment:
Minor nitpick: CB and cb_arg are in the wrong order here.
Could also use designated initializer list here